puppet/modules/web/templates/rpm/HEADER.html.epp

```diff
@@ -0,0 +1,66 @@
+<%- |
+  String $stable_foreman,
+  Stdlib::Fqdn $servername,
+| -%>
+<h1><%= $servername %></h1>
+
+<h3>Foreman</h3>
+
+<p>Foreman is available under <code>/releases/VERSION/DIST/ARCH</code>, e.g.</p>
+
+<ul>
+  <li>/foreman/<%= $stable_foreman %>/foreman/el9/x86_64</li>
+  <li>/foreman/nightly/foreman/el9/x86_64</li>
+</ul>
+
+<p>foreman-release RPMs containing an appropriate .repo file are available with fixed URLs:</p>
+
+<ul>
+  <li><a href="https://<%= $servername %>/foreman/<%= $stable_foreman %>/foreman/el9/x86_64/foreman-release.rpm">https://<%= $servername %>/foreman/<%= $stable_foreman %>/foreman/el9/x86_64/foreman-release.rpm</a></li>
+  <li><a href="https://<%= $servername %>/foreman/nightly/foreman/el9/x86_64/foreman-release.rpm">https://<%= $servername %>/foreman/nightly/foreman/el9/x86_64/foreman-release.rpm</a></li>
+</ul>
+
+<p>Release packages <a href="https://theforeman.org/security.html#GPGkeys">are signed with a new key for each major release</a>.  The public key is available in the RPM-GPG-KEY-foreman file within each version directory or the foreman-release RPMs.</p>
+
+<p>Nightly builds of Foreman are available under <code>/foreman/nightly/foreman/DIST/ARCH</code> and are refreshed a few times a day, but are not GPG signed.</p>
+
+<h3>Plugins</h3>
+
+<p>A number of Foreman plugins are available in the plugin repos, see <a href="https://projects.theforeman.org/projects/foreman/wiki/Plugins">Plugins</a> and <a href="https://projects.theforeman.org/projects/foreman/wiki/List_of_Plugins">List of Plugins</a> for more information.</p>
+
+<p>Plugin repos are structured by the Foreman version that they're compatible with in the format <code>/foreman/VERSION/plugins/DIST/ARCH</code>, e.g.</p>
+
+<ul>
+  <li>/foreman/<%= $stable_foreman %>/plugins/el9/x86_64</li>
+  <li>/foreman/nightly/plugins/el9/x86_64</li>
+</ul>
+
+<p>Plugin repos are not GPG signed.</p>
+
+<h3>Katello</h3>
+
+<p>Katello is available under <code>/foreman/VERSION/katello/DIST/ARCH</code> with Candlepin under <code>/candlepin/CANDLEPIN_VERSION/DIST/ARCH</code>.
+
+<p>katello-repos RPMs containing an appropriate .repo file are available:</p>
+
+<ul>
+  <li><a href="https://<%= $servername %>/foreman/3.11/katello/el9/x86_64/katello-repos-latest.rpm">https://<%= $servername %>/foreman/3.11/katello/el9/x86_64/katello-repos-latest.rpm</a></li>
```

Comment:
With the new layout we can solve an issue that current yum.theforeman.org has, which is that the version numbers no longer line up.
```suggestion
  <li><a href="https://<%= $servername %>/foreman/<%= $stable_foreman %>/katello/el9/x86_64/katello-repos-latest.rpm">https://<%= $servername %>/foreman/<%= $stable_foreman %>/katello/el9/x86_64/katello-repos-latest.rpm</a></li>
```